src/operator/nn/dnnl/dnnl_rnn.cc

```diff
@@ -449,11 +482,19 @@ void DNNLRnnForward::SetNewDataMem(void* x,
   auto& cpu_engine      = CpuEngine::Get()->get_engine();
   dnnl_args_map_t& args = net_args_;
 
+  int src_dtype = dtype;
+  int dst_dtype = dtype;
+  if (param_.quantized) {
+    src_dtype = mshadow::kUint8;
```

Comment:
RNN does not support int8?

Reply:
Now it does - seems like int8 is supported since 2.3 (https://oneapi-src.github.io/oneDNN/v2.2/dev_guide_rnn.html / https://oneapi-src.github.io/oneDNN/v2.3/dev_guide_rnn.html)

New task can be opened for int8 implementation and checking if it is worth